### DISCUSSION TITLE
AppAuthHandler issue: Expected token_type to equal "bearer", but got None instead

### DISCUSSION THREAD

**Discussion by ravindra-s** [tweepy/tweepy#1185] [Questions] [answered]
Hi,

I am trying to access Appplication-only authentication. Following is the code:

```
import tweepy
import sys

auth = tweepy.AppAuthHandler(MY_API_KEY,MY_API_SECRET)

api = tweepy.API(auth)

if (not api):
    print("Authentication failed!")
    sys.exit(-1)
```

I am getting following error:
```
Traceback (most recent call last):
  File "tweet_crawler.py", line 4, in <module>
    auth = tweepy.AppAuthHandler(MY_API_KEY,MY_API_SECRET)
  File "/home/ravi/.local/lib/python3.6/site-packages/tweepy/auth.py", line 170, in __init__
    'but got %s instead' % data.get('token_type'))
tweepy.error.TweepError: Expected token_type to equal "bearer", but got None instead
```

What seems to be the issue? Any help is appreciated!

**Comment by Harmon758** [comment, marked as answer]
Normally, I'd recommend that you reset your credentials since you leaked them, but they seem to be invalid anyway.
Double check that you are using the proper consumer key and secret.

**Comment by ravindra-s** [comment]
I realized that immediately and generated new ones. Thanks.

**Comment by nagmat1** [comment]
> Normally, I'd recommend that you reset your credentials since you leaked them, but they seem to be invalid anyway.
> Double check that you are using the proper consumer key and secret.

I am getting the same error, How can I overcome it?! 
The error is :
Expected toke_type to equal "bearer" but got None instead

**Comment by Harmon758** [comment]
As I answered in what you quoted:
> Double check that you are using the proper consumer key and secret.
